index.html

```diff
@@ -23,32 +23,32 @@
     <body >
         <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-C6RzsynM9kWDrMNeT87bh95OGNyZPhcTNXj1NW7RuBCsyN/o0jlpcV8Qyq46cDfL" crossorigin="anonymous"></script> <!-- Bootstrap-Source.  Place the <script> tag for our JavaScript bundle (including Popper for positioning dropdowns, poppers, and tooltips) before the closing </body>.-->
         
-        <div class="themeSelectContainer">            
-            <input type="radio" class="theme-switch" name="theme-switch" id="theme-switch1">
+        <div class="themeSelectContainer" aria-label="Téma választó">            
+            <input type="radio" class="theme-switch" name="theme-switch" id="theme-switch1" aria-label="Egyes téma kiválasztása" role="button"> <!-- A HTML validator ERROR-t dob, ha itt a role="button" szerepel: Bad value 'radio' for attribute 'type' on element 'input'. Marpedig szerintem ebben a specialis esetben a radio-input button role-ban van. Mi ilyenkor a teendö? :) -->
```

Comment:
Ezt nekem is kidobta de én sem értem pontosan az okát.. lehet nem mindig lehetett de szerintem is most lehet..

Reply:
Az inputok implicit hordoznak role-t, ezesetben a radio típusú input szintén radio role-lal fog rendelkezni alapból. Nem szükséges kiírni, viszont ha button-nak van álcázva, akkor hibás is.